source/GameObjects.h

```diff
@@ -0,0 +1,113 @@
+/* GameObjects.h
+Copyright (c) 2021 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef GAME_OBJECTS_H_
+#define GAME_OBJECTS_H_
+
+#include "CategoryTypes.h"
+#include "Sale.h"
+#include "Set.h"
+#include "Trade.h"
+
+#include "Color.h"
+#include "Conversation.h"
+#include "Effect.h"
+#include "Fleet.h"
+#include "Galaxy.h"
+#include "Government.h"
+#include "Hazard.h"
+#include "Interface.h"
+#include "Minable.h"
+#include "Mission.h"
+#include "News.h"
+#include "Outfit.h"
+#include "Person.h"
+#include "Phrase.h"
+#include "Planet.h"
+#include "Ship.h"
+#include "StartConditions.h"
+#include "System.h"
+#include "Test.h"
+#include "TestData.h"
+
+#include <future>
+#include <map>
+#include <string>
+#include <vector>
+
+
+class Sprite;
+
+
+
+// Class representing every game object.
+class GameObjects {
+public:
+	Set<Color> colors;
```

Comment:
If someday we use this class for tests then we'll need to have public access to it. So that's why I just made the class public so avoid a potential refactor later.